gravitee-am-management-api/gravitee-am-management-api-service/src/main/java/io/gravitee/am/management/service/impl/EmailServiceImpl.java

```diff
@@ -261,7 +261,7 @@ private Map<String, Object> prepareEmailParams(Domain domain, Application client
         }
 
         params.put("url", domainService.buildUrl(domain, redirectUri, queryParam));
-
+        params.put("gatewayHost", domainService.gatewayUrl().getHost().replaceFirst("^www\\.", ""));
```

Comment:
**question**: As it is not possible to use variable in the template for the "from" header, should we keep this variable :thinking: ? See https://github.com/gravitee-io/gravitee-access-management/pull/3275/files#diff-18fe570831b203224bdfd7400439c0778b685600a6ac699c057031f89c5c20afR13